app/src/main/res/layout-sw600dp/story_fragment.xml

```diff
@@ -36,13 +37,20 @@
         app:navigationContentDescription="@string/navigate_up"
         app:navigationIcon="?attr/homeAsUpIndicator">
 
+        <asia.ivity.android.marqueeview.MarqueeView
+          android:id="@+id/marquee_view"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          marquee:speed ="10"
```

Comment:
If it's not too much effort, could we maybe rename this & the next custom attributes in the base library to be something more understandable? Specifically:
- 'speed' of what?
- What are the units for 'speed' and 'pause'?
- Is 'pause' really just a 'delay'?

Including these details in the attribute names should help with clarity.

Reply:
Noted.